test/Wrappers/wrapper-erc721.test.ts

```diff
@@ -0,0 +1,72 @@
+import {ethers} from "hardhat"
+import { expect } from "chai"
+import { ERC721Wrapper } from "../../lib/wrappers/tokens/erc721"
+import { ERC721Mock } from "../../typechain-types"
+
+describe("ERC721 Wrapper", async () => {
+    let tokenContract: ERC721Mock
+    let tokenWrapper: ERC721Wrapper
+
+    beforeEach(async () => {
+        const [deployer] = await ethers.getSigners()
+
+        tokenContract = await ethers.deployContract("ERC721Mock", [deployer, "Mock", "MKT"])
+        tokenWrapper = new ERC721Wrapper(
+            tokenContract.target,
+            ethers.provider
+        )
+    })
+
+    it("should show token symbol", async () => {
+        expect(await tokenWrapper.symbol())
+            .to.be.equal("MKT")
+    })
+    it("should show token name", async () => {
+        expect(await tokenWrapper.name())
+            .to.be.equal("Mock")
+    })
+    it("Should show tokenUri", async () => {
+        expect(await tokenWrapper.tokenUri('1'))
+            .to.be.equal("")
+    })
+    it("should transferFrom", async () => {
+        const [deployer, target] = await ethers.getSigners()
+
+        expect(await tokenWrapper.balanceOf(await deployer.getAddress()))
+            .to.be.equal("1")
+        expect(await tokenWrapper.balanceOf(await target.getAddress()))
+            .to.be.equal("0")
+
+        expect(await tokenWrapper.withSigner(deployer).transferFrom(await deployer.getAddress(), await target.getAddress(), "1"))
+            .to.changeTokenBalance(tokenContract, [deployer, target], [-1, 1])
+    })
+    it("should approve and approveAll and transferFrom", async () => {
```

Comment:
Try keeping tests as atomic as possible. An ideal test has only one `expect` statement.

Reply:
Fixed in b6c571d